code/game/objects/items/AI_modules.dm

```diff
@@ -357,8 +365,36 @@ AI MODULES
 		law_datum.clear_zeroth_law(0)
 	..()
 
+/obj/item/ai_module/core/full/handle_unique_ai()
+	var/datum/ai_laws/unique_ai_laws =  GLOB.round_default_lawset
+	if(law_id == initial(unique_ai_laws.id))
+		return
+	return QDEL_MODULE
```

Comment:
consider just returning TRUE/FALSE, no need for a new define here

Reply:
I think it works nicely. I didn't want to copy INIT HINT QDEL ones but I think true doesn't really describe what it will mean. Mothblocks will have a very long rant about it if you ask

Reply:
If you change the name of the function, it will be.
Example:
should_qdel_module